### ISSUE TITLE
Bug in channel, has posts but shows none

### ISSUE THREAD

**Issue by sebsel** [EdwardHinkle/indigenous-ios#237]
I just had a crash and thought: o no, now I am obliged to tell Eddie.

So when I open my 'Vim' channel, which has 104 items according to the badge, I get this empty screen:
![img_1045](https://user-images.githubusercontent.com/16517999/47280494-ee263880-d5d6-11e8-99ea-097b92d66bf1.png)

When I click on 'Load More Posts', I get a 'No More Posts', and then the app crashes roughly a second later.

**Comment by sebsel** [comment]
Aahh, I already found out what is wrong: I deleted this Vim-channel because it was following one feed that turned out not to be very interesting. So it might be related to https://github.com/EdwardHinkle/indigenous-ios/issues/229, the channel cache.

**Comment by EdwardHinkle** [comment]
Yep, that seems to be the same issue! Thanks for reporting it. I’ll mark this one as a duplicate. I hope to have this fixed in the next version!
